### DISCUSSION TITLE
Can GitHub Copilot be used without using GitHub for source control?

### DISCUSSION THREAD

**Discussion by NSouth** [community/community#52110] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Could a company use GitHub Copilot for Business (with Copilot X when available) to aid in the coding process, but use a different provider for source control? I'm interested in Copilot but am not ready to migrate repositories to a new provider. How tied to other GitHub services is Copilot (X)? Thanks!

**Comment by meppu** [comment, marked as answer]
You can use GitHub Copilot without version control system. So yes, for example you can use GitLab for your remote repository and GitHub Copilot for programming if you are eligable to use.

**Reply by meppu** [reply]
Note to mention, you and your team must have a GitHub account and eligable to use GitHub Copilot (X)

**Reply by Anthonia-alexis** [reply]
How can one be eligible for github copilot?

**Reply by davecheney** [reply]
Please visit https://copilot.github.com and choose a subscription plan.

**Comment by amirinia** [comment]
if we use Gitlab or Azure DevOps and we have Github accounts, can GitHub Copilot (X) generate a description for Gitlab or Azure DevOps PR?
